clang/test/CheckedCRewriter/refarrsubscript.c

```diff
@@ -13,7 +13,8 @@ int **func(int **p, int *x) {
 
 struct foo { int **b; int n; };
 	//CHECK_NOALL: struct foo { int **b; int n; };
-	//CHECK_ALL: struct foo { _Array_ptr<_Ptr<int>> b; int n;   };
+	//CHECK_ALL: struct foo { _Array_ptr<_Ptr<int>> b;
+	//CHECK_ALL: int n;
```

Comment:
This line break is a little unfortunate. This is not a multi-decl since both the array pointer and the int have their own types. Is there a way to preserve the formatting in this case?

Reply:
This is surprising. The `isSingleDeclaration` method specifically checks for multiple declarations on the same line, not just  in the same statement.  I'm not sure why this is done, but it might be because of limitations in the old rewriting process. I will see if the check can be modified to treat declarations such as `int *a; int b` as separate declarations.